packages/lesswrong/components/posts/PostsList2.tsx

```diff
@@ -154,12 +154,13 @@ const PostsList2 = ({
   useOnMountTracking({eventType: "postList", eventProps: {postIds, hidePosts}, captureOnMount: eventProps => eventProps.postIds.length, skip: !postIds.length||loading})
 
   if (!orderedResults && loading) return <Loading />
+  if (results && !results.length && !showNoResults) return null
```

Comment:
Have you tested at least one other location in the site where showNoResults is used?

Reply:
Yeah, it's only used in a couple places - I tested and they looked unaffected to me